src/components/UserNicknamesPanel.js

```diff
@@ -60,7 +60,7 @@ class UserNicknamesPanel extends React.Component {
     const {
       user,
     } = this.props
-    const maxNicksReached = (user.relationships.nicknames?.data.length >= MAXNICKS)
+    const maxNicksReached = (user.relationships.nicknames.data?.length >= MAXNICKS)
```

Comment:
as mentioned before, user should still be considered nullable. while technically it shouldn't it's worth being careful about it because of page change edge cases.